ompi/mca/topo/treematch/configure.m4

```diff
@@ -34,13 +35,6 @@ AC_DEFUN([MCA_ompi_topo_treematch_CONFIG], [
     ompi_topo_treematch_libdir=
     ompi_topo_treematch_happy=0
 
-    AC_MSG_CHECKING([for hwloc support])
-    AS_IF([test $OPAL_HAVE_HWLOC -eq 0],
-          [AC_MSG_RESULT([no (treematch requires hwloc)])
-           ompi_topo_treematch_happy=0],
-          [AC_MSG_RESULT([yes])
-           ompi_topo_treematch_happy=1])
-
     AS_IF([test $ompi_topo_treematch_happy -eq 1],
```

Comment:
i gues this test should be removed since ompi_topo_treematch_happy cannot/should be 1 any more